core-android/src/main/java/com/uber/sdk/android/core/auth/LoginActivity.java

```diff
@@ -152,28 +190,44 @@ protected void init() {
     }
 
     protected void loadUrl() {
-        sessionConfiguration = (SessionConfiguration) getIntent().getSerializableExtra(EXTRA_SESSION_CONFIGURATION);
-        if (sessionConfiguration == null) {
-            onError(AuthenticationError.UNAVAILABLE);
+        Intent intent = getIntent();
+
+        sessionConfiguration = (SessionConfiguration) intent.getSerializableExtra(EXTRA_SESSION_CONFIGURATION);
+        responseType = (ResponseType) intent.getSerializableExtra(EXTRA_RESPONSE_TYPE);
+
+        if (!validateRequestParams()) {
             return;
         }
 
-        if ((sessionConfiguration.getScopes() == null || sessionConfiguration.getScopes().isEmpty())
```

Comment:
Why are we removing the scope check?

Reply:
That logic is now part of the helper method `validateRequestParams`